tests/Fake/ServiceFake.php

```diff
@@ -12,57 +10,66 @@
 class ServiceFake implements IndexingInterface, BatchDocumentRemovalInterface
 {
 
-    public $shouldError = false;
+    public bool $shouldError = false;
 
-    public $documents = [];
+    public array $documents = [];
 
-    public $maxDocSize = 1000;
+    public int $maxDocSize = 1000;
 
-    public function addDocument(DocumentInterface $item): IndexingInterface
+    public function addDocument(DocumentInterface $item): ?string
     {
         $this->documents[$item->getIdentifier()] = DocumentBuilder::singleton()->toArray($item);
-        return $this;
+
+        return $item->getIdentifier();
     }
 
-    public function addDocuments(array $items): BatchDocumentInterface
+    public function addDocuments(array $documents): array
     {
-        foreach ($items as $item) {
+        $ids = [];
+
+        foreach ($documents as $item) {
             $this->addDocument($item);
+            $ids[] = $item->getIdentifier();
```

Comment:
Can we combine these methods now? Or perhaps just skip the line above?

```suggestion
            $ids[] = $this->addDocument($item);
```

Reply:
https://github.com/silverstripe/silverstripe-search-service/pull/76/commits/b202ae856b1feb8b9a18a16909a817c09f345f97